src/Raven.Studio/wwwroot/Content/css/bootstrap/variables-body.less

```diff
@@ -815,3 +815,6 @@
 @database-settings-dirty-item-color: @color-2-2;
 @database-settings-invalid-item-color: @color-1;
 @database-origin-key-color: @color-2-2;
+
+@copy-button-color: @gray-light;
```

Comment:
why new definions? Can't we use existing (for example we have copy on document edit page) - it those copy styles are scoped, then we should move them to global scope and reuse.

Reply:
done - refactored so that all places use the same global definition